src/Main/DeathRecapTracker.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Abilities from 'Parser/Core/Modules/Abilities';
+import DEFENSIVE_BUFFS from 'common/DEFENSIVE_BUFFS';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import Tab from 'Main/Tab';
+import DeathRecap from './DeathRecap';
+
+class DeathRecapTracker extends Analyzer {
+
+  deaths = [];
+  events = [];
+  healed = [];
+  damaged = [];
+  cooldowns = [];
+  buffs = [];
+  lastBuffs = [];
+
+  static dependencies = {
+    combatants: Combatants,
+    abilities: Abilities,
+    spellUsable: SpellUsable,
+  };
+
+  on_initialized() {
+    this.cooldowns = this.abilities.abilities.filter(e => 
```

Comment:
This would read so much better if the logic for the filter was extracted, e.g.

```js
const hasCooldown = ability =>  (ability.category === Abilities.SPELL_CATEGORIES.DEFENSIVE || ability.category === Abilities.SPELL_CATEGORIES.SEMI_DEFENSIVE) && ability.enabled === true;

this.cooldowns = this.abilities.abilities.filter(hasCooldown);
```